R/mergeCheck.R

```diff
@@ -39,13 +39,10 @@ variant_limit = 100000
 tbl_size_limit = 500
 
 #testing = as.logical(args[1])
-if (testing) {
-  databaseSynIdMappingId = 'syn11600968'
-} else if (staging) {
-  databaseSynIdMappingId = 'syn12094210'
-} else {
-  databaseSynIdMappingId = 'syn10967259'
-}
+databaseSynIdMappingId <- get_database_to_synapse_mapping_synid(
```

Comment:
Nit: Thanks for adding this into a function, I would say that this is actually a hack and should probably use project_id to obtain the dataBaseSynIdMapping.  However, that probably requires a whole configuration change across the board so I'd leave things as is.